### PR TITLE
Fix gcc 11 warnings

### PR DESCRIPTION
**Pull request by suburbski**

### CODE REVIEW

File: src/cgame/cg_atmospheric.cpp [hunk 1]
```diff
@@ -356,7 +356,7 @@ static void CG_RainParticleRender(cg_atmosphericParticle_t *particle)
 	vec3_t     forward, right;
 	polyVert_t verts[3];
 	vec2_t     line;
-	float      len, frac, dist;
+	float      len, /*frac,*/ dist;
```
Comment (haapanen) [inline src/cgame/cg_atmospheric.cpp:359]: If frac is not used, why not just remove it instead?

File: src/cgame/cg_character.cpp [hunk 2]
```diff
@@ -162,7 +162,7 @@ static void CG_CalcMoveSpeeds(bg_character_t *character)
 	int           i, j, k;
 	float         totalSpeed;
 	int           numSpeed;
-	int           lastLow, low;
+	int           /*lastLow,*/ low;
```
Comment (haapanen) [inline src/cgame/cg_character.cpp:165]: Same here, why not just remove this?

File: src/cgame/cg_debriefing.cpp [hunk 3]
```diff
@@ -1245,7 +1245,7 @@ void CG_Debriefing_ChatEdit_Draw(panel_button_t *button)
 
 void CG_Debriefing_ChatBox_Draw(panel_button_t *button)
 {
-	int    w, h;
+	int    w /*, h*/;
```
Comment (haapanen) [inline src/cgame/cg_debriefing.cpp:1248]: Here as well.

File: src/cgame/cg_draw.cpp [hunk 4]
```diff
@@ -852,7 +852,7 @@ CG_DrawTeamInfo
 */
 static void CG_DrawTeamInfo(void)
 {
-	int w, h;
+	int w /*, h*/;
```
Comment (haapanen) [inline src/cgame/cg_draw.cpp:855]: Here as well.

File: src/cgame/cg_draw.cpp [hunk 5]
```diff
@@ -261,7 +261,7 @@ int CG_DrawFieldWidth(int x, int y, int width, int value, int charWidth, int cha
 {
 	char num[16], *ptr;
 	int  l;
-	int  frame;
+	// int  frame;
```
Comment (haapanen) [inline src/cgame/cg_draw.cpp:264]: Here as well.

File: src/cgame/cg_draw.cpp [hunk 6]
```diff
@@ -5208,8 +5208,10 @@ static void CG_DrawPlayerStatus(void)
 	if (etj_HUD_weaponIcon.integer)
 	{
 		// Draw ammo
-		// weap = CG_PlayerAmmoValue(&value, &value2, &value3);
 		value = value2 = value3 = 0;
+		// weap = 
+		CG_PlayerAmmoValue(&value, &value2, &value3);
```
Comment (haapanen) [inline src/cgame/cg_draw.cpp:5213]: All of these were previously commented out but now uncommented. Is this intentional or am I missing something?

Comment (suburbski) [reply src/cgame/cg_draw.cpp:5191]: I had commented them out accidentally because of unused return value, should be uncommented

**Pull request merged**